config/plugins/interactive_environments/ipython/templates/ipython.mako

```diff
@@ -10,6 +11,13 @@ ie_request.load_deploy_config()
 ie_request.attr.docker_port = 6789
 # Create tempdir in galaxy
 temp_dir = ie_request.temp_dir
+if ie_request.attr.viz_config.getboolean("docker", "wx_tempdir"):
```

Comment:
Please move this to `lib/galaxy/web/base/interactive_environments.py`, perhaps around https://github.com/scholtalbers/galaxy/blob/dev/lib/galaxy/web/base/interactive_environments.py#L53